index.html

```diff
@@ -37,11 +37,12 @@ <h1 class="intro-text">Open Source Design</h1>
     <div class="row">
         <div class="col-md-4">
             <h2>Articles</h2>
-            {% for post in site.posts limit:3 %}
+            {% for post in site.posts limit:4 %}
```

Comment:
As @dmichl said we should keep it at 3
```suggestion
            {% for post in site.posts limit:3 %}
```